templates/web/base/waste/services.html

```diff
@@ -47,6 +47,11 @@
          [% IF c.cobrand.moniker == 'kingston' AND (slwp_garden_sacks AND unit.garden_sacks) %]
              <input type="submit" value="Order more garden sacks" class="waste-service-descriptor waste-service-link">
          [% ELSE %]
+            [% IF c.cobrand.moniker == 'brent' %]
```

Comment:
This is only reached if `show_garden_modify` is true, which for Brent is only for staff (that's why we want to show this message), so this isn't shown to normal users (see normal user display attached). So it needs to be its own section, presumably can still be inside the `NOT pending_cancellation` section, but outside this section.

<img width="765" alt="image" src="https://github.com/mysociety/fixmystreet/assets/154364/9dc97ad4-2988-4308-b255-292bf876688c">